src/Spline.cpp

```diff
@@ -200,10 +200,25 @@ double base::geometry::SplineBase::getCurveLength(double relative_resolution) co
     return length;
 }
 
-double SplineBase::getUnitParameter()
```

Comment:
It was a constant factor before and still is one. Which is wrong... One could multiply it with 0.1 to make it finer grained or something. Still the result may be unexpected. 

Agreed, the method is broken (and was before) but this patchset does not aim at fixing it.